content/en/aws/kms/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Key Management Service (KMS)"
+linkTitle: "Key Management Service (KMS)"
+categories: ["LocalStack Community"]
+description: >
+  Key Management Service (KMS)
+---
+
+Key Management Service (KMS) is mainly about
+- creation and management of cryptographic keys
+- use of these keys to encrypt / decrypt data or to sign messages and to verify signatures for signed ones
+
+AWS documentation on KMS amd all its operations can be found [here](https://docs.aws.amazon.com/kms/latest/APIReference/Welcome.html).
+
+Keys in KMS roughly fall into the 3 categories:
+- Asymmetric encryption keys - holding a pair of a private and a public key. Asymmetric keys can be used either for data encryption and decryption with [Encrypt](https://docs.aws.amazon.com/kms/latest/APIReference/API_Encrypt.html) / [Decrypt](https://docs.aws.amazon.com/kms/latest/APIReference/API_Decrypt.html) or for creation and verification of digital signatures for messages with [Sign](https://docs.aws.amazon.com/kms/latest/APIReference/API_Sign.html) / [Verify](https://docs.aws.amazon.com/kms/latest/APIReference/API_Verify.html) operations. The same asymmetric key can't be used for both pairs of operations at the same time - when you create such a key, you have to specify in its [KeyUsage](https://docs.aws.amazon.com/kms/latest/APIReference/API_CreateKey.html#KMS-CreateKey-request-KeyUsage) parameter, which pair of operations the key will support.
+- Symmetric encryption keys - such keys consist of just one key. Symmetric keys can **not** be used to sign and to verify messages, are only used to encrypt and to decrypt data.
+- Hash-Based Message Authentication Code (HMAC) keys, a special case of symmetric keys that are only used in creation and verification of message authentication codes (MACs) with [GenerateMac](https://docs.aws.amazon.com/kms/latest/APIReference/API_GenerateMac.html) / [VerifyMac](https://docs.aws.amazon.com/kms/latest/APIReference/API_VerifyMac.html) operations. You can read more about this type of keys [here](https://docs.aws.amazon.com/kms/latest/developerguide/hmac.html).
+
+You can check [this page](https://docs.aws.amazon.com/kms/latest/developerguide/symm-asymm-compare.html) to see what other operations are supported for each type of keys. When you know what operations you require, this reference can help to figure out what key types are suitable for your goals.
+
+# Tutorial
+
+Let's create a simple symmetric encryption key and use it to encrypt / decrypt something.
+
+A new key can be created in KMS with
+
+```sh
```

Comment:
nit: would be better to use the `<command>` macro here, and prefix the command with `$`. (there's a few examples in other docs..)

Reply:
Done.